cylc/flow/scripts/scan.py

```diff
@@ -212,6 +212,16 @@ def get_option_parser():
         action='store_true'
     )
 
+    parser.add_option(
+        '--ping',
+        help=(
+            'Test the connection to the flow.'
+            ' (scanning will use the filesystem where possible, --ping forces'
+            ' a connection to the flow)'
```

Comment:
```suggestion
            'Test the connection to the flow. Scan normally just reads flow contact'
            ' files, but --ping forces a connection to the scheduler and removes the'
            ' contact file if it is not found to be running (this can happen if the'
            ' scheduler gets killed and can't clean up after itself).'
```
(Best to be explicit about what's going on?).